.github/workflows/test-multiple-versions.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
         with:
-          node-version: '12'
+          node-version: '14'
```

Comment:
what would happen with node 12?

Reply:
There's no fs/promises which many packages nowadays use!

Reply:
I see. We don't use fp/promises. What uses them??

Reply:
typed-macro!